release/models/bgp/openconfig-bgp-common.yang

```diff
@@ -696,21 +702,57 @@ submodule openconfig-bgp-common {
 
   grouping bgp-common-use-multiple-paths-link-bandwidth-config {
     description
-      "Parameters controlling usage of  of DMZ Link-Bandwidth
-      extended community in pultipath RIB/FIB formation";
+      "Parameters controlling usage of DMZ Link-Bandwidth
+      extended community in multipath RIB/FIB formation";
+
     leaf enabled {
       type boolean;
       description
-      "When set to TRUE, BGP multiplepath shall distributed traffic
-      load among contributing routes proportionally to value of
-      Local Administrator subfield of link-bandwidth extended
-      community [draft-ietf-idr-link-bandwidth-07].
-      This leaf has no effect if BGP multi-path is disabled or
-      if maximum-path attribute of BGP multi-path value is set
-      to 1";
+        "When set to TRUE, BGP multiplepath shall distributed traffic
+        load among contributing routes proportionally to value of
+        Local Administrator subfield of link-bandwidth extended
+        community [draft-ietf-idr-link-bandwidth-07].
+        This leaf has no effect if BGP multi-path is disabled or
+        if maximum-path attribute of BGP multi-path value is set
+        to 1";
+    }
+  }
+
+  grouping bgp-common-link-bw-config {
+    description
+      "Configuration parameters relating to link-bandwidth for BGP.";
+    reference
+      "https://datatracker.ietf.org/doc/draft-ietf-bess-ebgp-dmz/05";
+
+    leaf send-cumulative {
+      type boolean;
+      description
+        "When set to TRUE, and sending of link-bandwidth community values
+        is enabled, the system will sum the bgp link-bandwidth community
+        values for equal cost paths as defined in
+        draft-ietf-bess-ebgp-dmz-05.";
+    }
+
+    leaf non-transitive-ebgp {
+      type boolean;
+      when "../send-cumulative = 'TRUE'";
+      description
+        "When set to TRUE and sending of link-bandwidth community values is
```

Comment:
Is this the correct description? Is this really sending a community that is marked as non-transitive and then making it transitive? My understanding was that this is for the _originator_ of the community to mark it transitive? @rszarecki